src/commands/copy.rs

```diff
@@ -51,8 +69,8 @@ impl CopyCmd {
     fn inner_run(&self) -> Result<()> {
         let config = RUSTIC_APP.config();
 
-        if config.copy.targets.is_empty() {
-            status_err!("no [[copy.targets]] section in config file found!");
+        if config.copy.target.is_empty() {
+            status_err!("please specify at least 1 target!");
```

Comment:
I think that error message should make the actual cause of the error more explicit.
Something along the lines:

"Copy target must be a standalone rustic profile, target {target} doesn't exist."

Reply:
Here, `target` is just a `String`, but I adapted the error message to be more clear.

Also, when the profile cannot be loaded, now an `error` log is displayed:
```
[INFO] using config /home/alex-dev/.config/rustic/default.toml
[INFO] repository local:/tmp/repo: password is correct.
[INFO] using cache at /home/alex-dev/.cache/rustic/cc2abcdea25343bcc6072cbeb3452fb6c735aea7682d1653a10b01e980703e6c
[00:00:00] reading index...               ████████████████████████████████████████          2/2                                                                             
[00:00:00] getting snapshots...           ████████████████████████████████████████          4/4                                                                             
[ERROR] using no config file, none of these exist: /home/alex-dev/.config/rustic/xxx:.toml, /etc/rustic/xxx:.toml, ./xxx:.toml
error: No repository given.
```